go/consensus/api/api.go

```diff
@@ -32,6 +32,12 @@ type ClientBackend interface {
 	// SubmitTx submits a signed consensus transaction.
 	SubmitTx(ctx context.Context, tx *transaction.SignedTransaction) error
 
+	// StateToGenesis returns the genesis state at the specified block height.
+	StateToGenesis(ctx context.Context, blockHeight int64) (*genesis.Document, error)
+
+	// WaitEpoch waits for consensus to reach an epoch.
+	WaitEpoch(ctx context.Context, epoch epochtime.EpochTime) error
```

Comment:
Not new, just moved from the client API as it makes more sense in consensus (or actually it would make even more sense in epochtime, maybe I should move it there).